### ISSUE TITLE
(Feature request) Option to remove ALL note revisions from the database

### ISSUE THREAD

**Issue by Br0kenSilos** [TriliumNext/trilium#5088]
### Describe feature

I think it would be nice to have a "one-click" option to clear the database of all previous note revisions.  This would be to simply shrink the DB and free up space when things are in a state that the user feels is safe to remove all revisions / snapshots. This would go a long way towards keeping the overall size of the DB (and its backups) down.

### Additional Information

Ideally the option to clear all revisions from the database would be available either at the "root" as a right-click option. Or perhaps a better place would be to locate it under Options > Advanced (near the vacuum db option) ) to remove ALL current Revision Snapshots. 

Clicking on the option should probably present the user with a clear and concise warning about what the option will do so that they understand that it is going to remove all previous note revisions "globally".

**Comment by eliandoran** [comment]
This is possible via:

1. Going Options -> Other -> Note Revision Snapshot Limit
2. Typing 0 (no revisions) instead of `-1`(no limit)
3. Pressing "Erase excess revision snapshots now".
4. Typing -1 back in (or the desired limit of snapshots).

**Comment by Br0kenSilos** [comment]
@eliandoran Yes. I know we can do it that way. And that's totally fine. I was just thinking more of a simpler way to clear all current revisions with a single button somewhere. Honestly I don't know how many people would be interested in such a thing. It might be the IT / OCD side of me just constantly cleaning up behind myself..  🤣

So in my case.. Lets assume I normally have my revisions set to 5. And I have ~5,000 notes with 2-5 revisions each. And lets also assume I am doing a cleanup where I am comfortable that the current version is all I need (for now).. I follow you process below.

1.  Going Options -> Other -> Note Revision Snapshot Limit
2.  Typing 0 (no revisions) instead of -1(no limit)
3.  Pressing "Erase excess revision snapshots now".
4.  Typing 5 back in (or the desired limit of snapshots).

This works totally fine. But I was thinking... Just leave all the settings alone and have a little button or something to just clear what is currently there, like a temporary override and clear all in one step. Granted, there probably isn't a huge "need" or desire for that as a feature. And one could argue that you could just add some text on that screen of how to accomplish the process as you have laid out.
